presto-docs/src/main/sphinx/functions/string.rst

```diff
@@ -118,6 +118,13 @@ String Functions
     ``entryDelimiter`` splits ``string`` into key-value pairs. ``keyValueDelimiter`` splits
     each pair into key and value.
 
+.. function:: split_to_multimap(string, entryDelimiter, keyValueDelimiter) -> map<varchar, array<varchar>>
+
+    Splits ``string`` by ``entryDelimiter`` and ``keyValueDelimiter`` and returns a map,
+    where the value is an array, in the same order as they were in the ``string``.
```

Comment:
I am not sure if we want to document the ordering. cc @electrum

Reply:
The ordering seems fairly important. I don't know why we wouldn't specify the behavior. Add a comment in the test the mentions the ordering (so it's clear to anyone reading or changing that this is intended and not just an implementation detail)